.hintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
### Your readme

- [ ] You didn't add a descriptive readme for your project. :fearful: Please add it because it is the documentation of your project :+1: You can add it using [microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md) :blush:
- [ ] You didn't add an MIT.md file which is the license file of your application. :+1: Please add it using this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) :+1:
- [ ] After adding your MIT.md file don't forget to add your name, current year, app-type in your MIT.md file because most of the students forget :thinking: